### ISSUE TITLE
Docs: How to create a new version

### ISSUE THREAD

**Issue by gplssm** [openego/ego.io#56]
Describe how to use a the sqlacodegen script to create a new version that suits a newly create _Data processing_ version.

**Comment by nesnoj** [comment]
Ähm, u mean something like this?
https://github.com/openego/ego.io/wiki/How-to-update-database-definitions

**Comment by gplssm** [comment]
Exactly! We decided to provide some documentation for ego.io. This includes one page for creating a new version. I will think about just leaving it in the wiki (as usual) and just referring to it.

**Comment by nesnoj** [comment]
Can we close this one @gplssm?
